services/packageManager.js

```diff
@@ -163,6 +163,29 @@ function parsePacmanQuery(packageName, output, returnObj, fulfill) {
   fulfill(returnObj);
 }
 
+function dpkgCrossReferenceInstallPackages(output, returnObj) {
+  let stdout = '';
+  output.text.forEach((outputObject) => {
+    if (outputObject.type === 'stdout') {
+      stdout += outputObject.data;
+    }
+  });
+  stdout = stdout.split(os.EOL);
+
+  stdout.forEach((line) => {
+    if (line !== '') {
+      const packageName = line.substr(4).split(' ')[0];
+      for (let i = 0; i !== returnObj.result.length; i += 1) {
```

Comment:
Yes, I wanted to break the loop as soon as `packageName`s match together, and `break` do not work in a `forEach()`